tests/Casts/IntegerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Laravel\Tests\Casts;
+
+use MongoDB\Laravel\Tests\Models\Casting;
+use MongoDB\Laravel\Tests\TestCase;
+
+class IntegerTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Casting::truncate();
+    }
+
+    public function testInt(): void
+    {
+        $model = Casting::query()->create(['intNumber' => 1]);
```

Comment:
A test with a float casted to `int` would be useful for exhautivity.

Reply:
Done.